integration_tests/go.mod

```diff
@@ -5,18 +5,22 @@ go 1.16
 require (
 	github.com/ninedraft/israce v0.0.3
 	github.com/pingcap/failpoint v0.0.0-20210316064728-7acb0f0a3dfd
-	github.com/pingcap/kvproto v0.0.0-20211122024046-03abd340988f
+	github.com/pingcap/kvproto v0.0.0-20211213085605-3329b3c5404c
 	github.com/pingcap/tidb v1.1.0-beta.0.20211229035549-783432895924
 	github.com/pingcap/tidb/parser v0.0.0-20211229035549-783432895924 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
 	github.com/tikv/client-go/v2 v2.0.0
-	github.com/tikv/pd v1.1.0-beta.0.20211118054146-02848d2660ee
+	github.com/tikv/pd v1.1.0-beta.0.20211214024235-3c626f28bd32
 	go.uber.org/goleak v1.1.12
 )
 
 replace github.com/tikv/client-go/v2 => ../
 
+replace github.com/pingcap/tidb v1.1.0-beta.0.20211229035549-783432895924 => github.com/lemonhx/tidb v1.1.0-beta.0.20220104042428-0243e66a282b
```

Comment:
why not use pingcap/tidb and pingcap/parser?

Reply:
Because use `pingcap/tidb` will cause the integration_test build failed.

Reply:
> Because use pingcap/tidb will cause the integration_test build failed.

why?

Reply:
Because `pingcap/tidb` doesn't implements the latest `pdClient` interface. So need to replace it, otherwise, the test will build failed.
```
pingcap/client-go/integration_tests  405 ✗                                                                                                                                                                                         
▶ go test
# github.com/pingcap/tidb/store/mockstore/unistore
../../../../../pkg/mod/github.com/pingcap/tidb@v1.1.0-beta.0.20211229035549-783432895924/store/mockstore/unistore/mock.go:71:2: cannot use pdClient (type *pdClient) as type "github.com/tikv/pd/client".Client in return argument:
        *pdClient does not implement "github.com/tikv/pd/client".Client (missing LoadGlobalConfig method)
../../../../../pkg/mod/github.com/pingcap/tidb@v1.1.0-beta.0.20211229035549-783432895924/store/mockstore/unistore/pd.go:28:5: cannot use new(pdClient) (type *pdClient) as type "github.com/tikv/pd/client".Client in assignment:
        *pdClient does not implement "github.com/tikv/pd/client".Client (missing LoadGlobalConfig method)
FAIL    integration_tests [build failed]

```

Reply:
Typically, we merge the PR that updates the mockstore in TiDB first.

Reply:
> pingcap/tidb doesn't implements the latest pdClient interface

why? should you update `"github.com/tikv/pd/client".Client` to implement the `pdClient` interface?

Reply:
first I need to pass compile